internal/pkg/skuba/deployments/ssh/cri.go

```diff
@@ -114,3 +117,47 @@ func criStart(t *Target, data interface{}) error {
 	_, _, err := t.ssh("systemctl", "enable", "--now", "crio")
 	return err
 }
+
+// CriMigrate migrates the old configuration of cri-o < 1.17 to the new format.
+func CriMigrate() error {
+	if err := criGenerateLocalConfiguration(); err != nil {
```

Comment:
the `deployments ssh cri.go` is supposed to happen on the node. This should not touch the local configuration file on the management node.

I don't think it's a good idea to have this function in here.